<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登陆</string>
  </property>
  <widget class="QAxWidget" name="axWidget_AgentOCX">
   <property name="control" stdset="0">
    <string>{781a3be7-557f-4d0d-af8d-4d8d798692c2}</string>
   </property>
   <property name="geometry" stdset="0">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>90</y>
     <width>195</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelAgentID">
          <property name="text">
           <string>坐席工号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lableAgentName">
          <property name="text">
           <string>坐席名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelThisDN">
          <property name="text">
           <string>分机号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAgentGroup">
          <property name="text">
           <string>坐席组</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditAgentID">
          <property name="text">
           <string>2005</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAgentName">
          <property name="text">
           <string>zwl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditThisDN">
          <property name="text">
           <string>1005</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxAgentGroup"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonLogin">
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
